[CI] ClusterRequestTests testFallbackIndicesOptions failing

**Build Scans:**
- [elasticsearch-periodic #5247 / single-processor-node-tests](https://gradle-enterprise.elastic.co/s/kwo4p5u2nj4te)
- [elasticsearch-periodic #5247 / concurrent-search-tests](https://gradle-enterprise.elastic.co/s/vh7pty3diwwa6)
- [elasticsearch-periodic #5247 / openjdk22_checkpart3_java-matrix](https://gradle-enterprise.elastic.co/s/syb2uipvpqihy)
- [elasticsearch-periodic-platform-support #5220 / rocky-8_platform-support-unix](https://gradle-enterprise.elastic.co/s/ujajsaizpdk2q)
- [elasticsearch-periodic-platform-support #5220 / oraclelinux-7_platform-support-unix](https://gradle-enterprise.elastic.co/s/yqfipaphknbv2)
- [elasticsearch-intake #13876 / part3](https://gradle-enterprise.elastic.co/s/wgxqh3m5x5kgo)
- [elasticsearch-intake #13852 / part3](https://gradle-enterprise.elastic.co/s/6xrljyp3cyr46)
- [elasticsearch-pull-request #44498 / part-3](https://gradle-enterprise.elastic.co/s/fue62rnbmo77w)
- [elasticsearch-pull-request #44401 / part-3](https://gradle-enterprise.elastic.co/s/inrixhxd533dy)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:test" --tests "org.elasticsearch.xpack.esql.plugin.ClusterRequestTests.testFallbackIndicesOptions" -Dtests.seed=40C7970CFE91DE92 -Dtests.configure_test_clusters_with_one_processor=true -Dtests.locale=nds-NL -Dtests.timezone=Antarctica/Troll -Druntime.java=23
```

**Applicable branches:**
8.16

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.plugin.ClusterRequestTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testFallbackIndicesOptions),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: <EsqlConfiguration{pragmas={"data_partitioning":"SHARD"}, resultTruncationMaxSize=651061541, resultTruncationDefaultSize=1386430430, locale=en_NA, query='from test
| sort last_name
| limit 10
| where round(emp_no) > 10
| eval c = first_name
', profile=false, tables={SNynvIxSR={xUV=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=6829, values=[-708571922, 702707664, -1227380550, 618180872, 162660347, -1146525606, -487458503, 959459401, -2019947437, -430932923, ...]]]], DmyBr=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=6829]]], Rbuy=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=6829, values=[-431318102, -1458312317, 319158037, 1873966889, 1962991840, -1292056016, 1611883941, 1210797763, 1810664293, 761270623, ...]]]], E=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=6829, values=[-1436207267, -941349889, 1045522256, -1666526844, -1638107104, 2100628798, 95198700, 2023929019, -451294962, -1632283410, ...]]]], Xm=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=6829, values=[1170265524, 122955217, -770907051, -196797167, -1301197370, -1344178133, -242363887, -1380021793, -1135170040, -1412622706, ...]]]], XlWrPs=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=6829, values=[-1311354456, -1582983150, -607092642, -1652885756, -51789148, -1631435585, -1360137727, -1061330274, 310265501, 1118318852, ...]]]]}, cQxUifUfCH={y=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=1274]]], oZ=Column[type=LONG, values=LongVectorBlock[vector=LongArrayVector[positions=1274, values=[713955223848599904, -4079201052244287083, 4392424971775464586, 6564127379595070581, -4449726321442071435, 3623265176374147415, -3085791037737576912, 4213463497642689056, -4458608507806804534, 1110109651240520511, ...]]]]}, LVPaq={UkbiVmmV=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=7758, values=[713197956, 733187054, -2105499307, 1578840240, -1640909815, 1850391975, -1615651676, 424215339, 1399168152, -949682258, ...]]]], E=Column[type=LONG, values=LongVectorBlock[vector=LongArrayVector[positions=7758, values=[3787622399019211459, 5243804304933509669, 7987156028699843479, 6723739134612902521, 5890497655005797543, -4034558937637455289, 4355950178138338325, -1552980898028381053, -8129933987725197904, -5356596421306510271, ...]]]], TawgJcA=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=7758, values=[440262981, -1619360257, -360419202, 775082449, -187218418, 1566136413, -404449722, 264898493, -738554646, 328154291, ...]]]], RVsNmtAXV=Column[type=LONG, values=LongVectorBlock[vector=LongArrayVector[positions=7758, values=[-4480693345165698719, 7399278582720212908, 2111388143224894636, 2582049117161804493, -4086271102533675873, -2848474911806821757, -613607909888836556, -8626644010810162566, 8350177197465851153, 4316967136474540272, ...]]]], sgXg=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=7758]]], DeG=Column[type=LONG, values=LongVectorBlock[vector=LongArrayVector[positions=7758, values=[1929761449109560429, 5798432577061002153, -522044247350780219, 4899828181390591383, 7553974386518460520, -36806971087740167, -8098639470286053331, -1787706579890868665, -3073184986723893673, 890360817616462314, ...]]]], ehShRW=Column[type=LONG, values=LongVectorBlock[vector=LongArrayVector[positions=7758, values=[-5282874468309080782, 7934256484075592307, 1377042093211684740, 2525664291342042028, -4720886037839233708, -4619444625552574146, 6628807437944232482, -33754826357730529, -6727182096683239638, 2925323455655375842, ...]]]], gq=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=7758, values=[-1748481904, 640721346, 1605413724, -2088862169, -1330483025, -1503119223, 21890356, -16888770, -958918843, 190535344, ...]]]], hHoJL=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=7758]]]}, IDN={vW=Column[type=LONG, values=LongVectorBlock[vector=LongArrayVector[positions=5995, values=[1628996902947805344, 601274202791530132, -7271227752248339442, -1266108703772056290, 1334036624844224021, -3751274421670632202, 7009534350865459809, -7409866090066002665, 1605714867861017040, -4199344342998474863, ...]]]], WaKRwNjzM=Column[type=LONG, values=LongVectorBlock[vector=LongArrayVector[positions=5995, values=[-438288758638255075, -4080381736608248861, -4681495298093382861, 6658431101866510979, 3494050534528338904, 1950693600536706812, 4445385190042135101, -143194844573035452, -2475138812577369826, -8414050522284875520, ...]]]], YFXicAyf=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=5995]]], cvI=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=5995, values=[-1712402236, -1360938471, 584706648, 400167030, -1470748839, 1120892057, 640814261, 119358885, 714662468, 1618871385, ...]]]], FwsHdp=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=5995]]], aopdkhG=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=5995, values=[-2146936587, 1046786230, 339240978, -295089162, -61379996, 201381630, 1875417679, -588745242, 94472728, 1274542016, ...]]]], coGQ=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=5995, values=[-562310706, -1638601509, 748841218, 62917562, -2010292714, 1852088552, 1843979952, 515952558, -1941939064, -680265910, ...]]]], JRDlC=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=5995]]], n=Column[type=LONG, values=LongVectorBlock[vector=LongArrayVector[positions=5995, values=[3769466291764361931, 1222149940454978062, -6975486605696070637, 6546075953678245420, 328255761582219414, -3457391700030129844, 8844058066868682490, 1000668714097362377, 3488634090564696250, 501010059066111826, ...]]]]}, Rf={MeS=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=6735, values=[-2034190089, 214993653, 1255546925, 1193715448, -1878043356, 1808066112, -1264772621, -219729074, 757497390, 436362398, ...]]]], V=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=6735, values=[-229278123, 213256380, 1354370193, 886821166, 1685374540, 651723096, -2004991624, 1101453149, 193705648, -684324859, ...]]]], LB=Column[type=LONG, values=LongVectorBlock[vector=LongArrayVector[positions=6735, values=[-4895083643654430097, 9167171720457296735, -8043595189414669145, -3697800320865283809, -2488619190748836004, -1560187055465059516, -100001752177734698, 2515449114785703502, -4366836847366538868, -2117982580291425189, ...]]]], VJSn=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=6735]]]}, inKnMqy={P=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=4275]]], petEeaI=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=4275]]], fCP=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=4275]]], XohL=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=4275]]], utPGH=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=4275, values=[-935144631, 110677516, 1904047549, -1882261641, -1306440416, 621241644, -599587566, 1746128363, -1222603670, -664873805, ...]]]], pK=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=4275, values=[1418007781, -186355980, 450525558, 1601521071, 1455926366, -814573801, 1733774439, 1772265269, -926370133, -1271513849, ...]]]], eCtxcM=Column[type=LONG, values=LongVectorBlock[vector=LongArrayVector[positions=4275, values=[-9056680954251116551, 966145393395168266, -7064880454211225403, -5641408253806966443, -7562620538197918879, -8436753779560301459, 69313608896789181, 4531256610465048102, -3339380675231855208, 681203766516691957, ...]]]]}, l={q=Column[type=LONG, values=LongVectorBlock[vector=LongArrayVector[positions=8702, values=[6059789432268084608, -4775520438655668527, 1434808271857936647, 3938441329809828446, -3798969594667288004, -8506481149181465327, -8024446169023315659, -1377206086111730888, 1770154595196056819, -951599212734040954, ...]]]], Oh=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=8702]]], XcRy=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=8702, values=[-704838093, -1317479433, 2072808655, 1547490373, 2053498446, 1000683130, -1241997493, 1952447017, -1190379181, 547633603, ...]]]], YIt=Column[type=LONG, values=LongVectorBlock[vector=LongArrayVector[positions=8702, values=[7555327246078820144, -8691971137578613413, 4559589735304410326, 2870517069568377707, 3077860625510110218, -3391190864238304923, 8888138837934869506, -4999262381088716491, 7866097463474394156, 9121280055267612908, ...]]]]}, KtEEQHsJ={tf=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=2065, values=[549163357, 505582223, 1330198284, 1855247396, 474552349, 1945827107, 1454514160, 1463357604, -1016357820, -1825947291, ...]]]], jELuP=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=2065]]], yFz=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=2065]]], M=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=2065, values=[-146664637, -1446102288, 1438950316, 774571043, -1695847019, 2052989030, 1275281431, 1757201520, 1566645986, -658193347, ...]]]], LyPDUG=Column[type=LONG, values=LongVectorBlock[vector=LongArrayVector[positions=2065, values=[-5564683985585142780, -3530908190193328183, -5043272124972203280, -7329440196907530425, -8815077264884960480, -3844771999020302445, 6004910598850268223, 2584734279088020559, 5485761646344727367, -4057864940391836416, ...]]]], aMYh=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=2065, values=[-979170215, 301757702, -1109442634, -801675059, -1668801636, 1911906830, 301816472, 1074610681, -1096542747, -373347879, ...]]]]}, mvKp={a=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=554, values=[-1406782452, -106196724, 630593356, 1994736977, -1999716345, -1269490683, 1674839907, -655256274, -1954583459, -519436718, ...]]]]}, wJqDQu={QA=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=2092, values=[-1356051696, 194170301, -1443385171, -450620478, 957648148, -201311676, 1470040324, -2087624385, 1578721535, 36552370, ...]]]], zbY=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=2092]]], NjRFxl=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=2092, values=[-97957457, -28789067, 1543040734, 589653096, -83911073, 1476574384, -1938831276, 1031368950, 1658879356, -242840518, ...]]]], hZSkrfq=Column[type=KEYWORD, values=BytesRefVectorBlock[vector=BytesRefArrayVector[positions=2092]]], V=Column[type=INTEGER, values=IntVectorBlock[vector=IntArrayVector[positions=2092, values=[1138421799, 381055798, 1738928155, 766413524, -1139770054, 747238023, -16436733, -1952896609, -1481848822, -1243322231, ...]]]], LFux=Column[type=LONG, values=LongVectorBlock[vector=LongArrayVector[positions=2092, values=[6453062447803207673, 3072611305103065854, -4779871287000630213, -3759656794489341552, -7880264949235405485, -1638946315929466064, -6038233710638630125, -5923257477782848321, -623177487517318315, -2289619405962143904, ...]]]], rKIoV=Column[type=LONG, values=LongVectorBlock[vector=LongArrayVector[positions=2092, values=[3492010409105429431, -2120001340631659898, 5708717609625633858, 7265233929414143447, 3406241151204935786, 9043441204336624074, 8950703952465358567, 4642054948797981971, 3274872735706061522, -8207443411946558714, ...]]]]}}}>
     but: was <EsqlConfiguration{pragmas={"data_partitioning":"SHARD"}, resultTruncationMaxSize=651061541, resultTruncationDefaultSize=1386430430, locale=en_NA, query='from test
| sort last_name
| limit 10
| where round(emp_no) > 10
| eval c = first_name
', profile=false, tables={}}>
```

**Issue Reasons:**
- [8.16] 9 failures in test testFallbackIndicesOptions (8.5% fail rate in 106 executions)
- [8.16] 2 failures in step part3 (12.5% fail rate in 16 executions)
- [8.16] 2 failures in step part-3 (16.7% fail rate in 12 executions)
- [8.16] 2 failures in pipeline elasticsearch-intake (12.5% fail rate in 16 executions)
- [8.16] 2 failures in pipeline elasticsearch-pull-request (16.7% fail rate in 12 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -122,9 +122,6 @@ tests:
 - class: org.elasticsearch.test.rest.yaml.CcsCommonYamlTestSuiteIT
   method: test {p0=search.highlight/50_synthetic_source/text multi unified from vectors}
   issue: https://github.com/elastic/elasticsearch/issues/117815
-- class: org.elasticsearch.xpack.esql.plugin.ClusterRequestTests
-  method: testFallbackIndicesOptions
-  issue: https://github.com/elastic/elasticsearch/issues/117937
 - class: org.elasticsearch.xpack.ml.integration.RegressionIT
   method: testTwoJobsWithSameRandomizeSeedUseSameTrainingSet
   issue: https://github.com/elastic/elasticsearch/issues/117805

--- a/x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/plugin/ClusterRequestTests.java
+++ b/x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/plugin/ClusterRequestTests.java
@@ -155,11 +155,14 @@ public class ClusterRequestTests extends AbstractWireSerializingTestCase<Cluster
 
     public void testFallbackIndicesOptions() throws Exception {
         ClusterComputeRequest request = createTestInstance();
-        var version = TransportVersionUtils.randomVersionBetween(random(), TransportVersions.V_8_14_0, TransportVersions.V_8_16_0);
-        ClusterComputeRequest cloned = copyInstance(request, version);
+        var oldVersion = TransportVersionUtils.randomVersionBetween(
+            random(),
+            TransportVersions.V_8_14_0,
+            TransportVersionUtils.getPreviousVersion(TransportVersions.V_8_16_0)
+        );
+        ClusterComputeRequest cloned = copyInstance(request, oldVersion);
         assertThat(cloned.clusterAlias(), equalTo(request.clusterAlias()));
         assertThat(cloned.sessionId(), equalTo(request.sessionId()));
-        assertThat(cloned.configuration(), equalTo(request.configuration()));
         RemoteClusterPlan plan = cloned.remoteClusterPlan();
         assertThat(plan.plan(), equalTo(request.remoteClusterPlan().plan()));
         assertThat(plan.targetIndices(), equalTo(request.remoteClusterPlan().targetIndices()));

